src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -137,8 +140,11 @@ export class EditorPart extends Part implements IEditorPart {
 	private centeredLayoutWidget!: CenteredViewLayout;
 
 	private gridWidget!: SerializableGrid<IEditorGroupView>;
+	private gridWidgetDisposables: DisposableStore = this._register(new DisposableStore());
 	private readonly gridWidgetView = this._register(new GridWidgetView<IEditorGroupView>());
 
+	private disposableMaximizedView: IDisposable = Disposable.None;
```

Comment:
```suggestion
	private disposableMaximizedView: IDisposable | undefined = undefined;
```

Reply:
We use this pattern in disposable2x2. Setting `Disposable.None` is the nicer option. We don't want to check for undefined and we shouldn't.

Reply:
I was more arguing that it is `undefined` when not maximized, but I do not have strong feelings. With using optional chaining `disposableMaximizedView?.dispose()` I really do not bother anymore about a type being `undefined`.